src/frontend/admin/src/components/presentational/table/TableDefaultActions.tsx

```diff
@@ -35,12 +36,14 @@ export function TableDefaultActions(props: Props) {
   const deleteModal = useModal();
 
   const menuItems: TableDefaultMenuItem[] = useMemo(() => {
+    const other = props.extendedOptions ?? [];
     return [
       {
         title: "Edit",
         icon: <EditIcon fontSize="small" />,
         onClick: props.onEdit,
       },
```

Comment:
In the case of accesses, edit is useless. The option is displayed but nothing is bound.